prefect_saturn/core.py

```diff
@@ -7,7 +7,6 @@
 from typing import Any, Dict, List, Optional, Union
 from requests import Session
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
```

Comment:
The removal of this line was due to an error I was getting in vscode:

```
Import "requests.packages.urllib3.util.retry" could not be resolved from source
```

I can remove the commit that does this. However `HTTPAdapter` takes an `int` and will create a `Retry` the same way we do so it isn't _needed_.